ocaml/idl/datamodel.ml

```diff
@@ -5984,7 +5984,8 @@ let pool_enable_ha = call
   ~in_oss_since:None
   ~params:[
     Set(Ref _sr), "heartbeat_srs", "Set of SRs to use for storage heartbeating.";
-    Map(String, String), "configuration", "Detailed HA configuration to apply"]
+    Map(String, String), "configuration", "Detailed HA configuration to apply";
+    String, "cluster_stack", "HA Cluster manager stack";]
```

Comment:
I realise that this is a design question, but would an Enum type not make sense here?

Also, it would be good to add some more information about this new parameter, most importantly when (in which release) it was introduced. To do that, you'd have to change `~params` into `~versioned_params`. Search in datamodel.ml for examples.